tests/queries/0_stateless/02352_grouby_shadows_arg.sql

```diff
@@ -0,0 +1,5 @@
+SET prefer_localhost_replica=0;
+SELECT toString(dummy) as dummy FROM remote('127.{1,1}', 'system.one') GROUP BY dummy;
+SELECT toString(dummy+1) as dummy FROM remote('127.{1,1}', 'system.one') GROUP BY dummy;
+SELECT toString((toInt8(dummy)+2) * (toInt8(dummy)+2)) as dummy FROM remote('127.{1,1}', system.one) GROUP BY dummy;
+SELECT round(number % 3) AS number FROM remote('127.{1,1}', numbers(20)) GROUP BY number ORDER BY number ASC;
```

Comment:
just FYI this query worked before the fix

Reply:
It did, just making sure we fix the desired behaviour in the test as well.